### PR TITLE
Add help info about price oracle

### PR DESCRIPTION
**Pull request by fabriziovigevani**
closes 1Hive/gardens#321.

Addresses changes done in which where done against master and includes PR comments

### CODE REVIEW

File: src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js [hunk 1]
```diff
@@ -392,6 +393,24 @@ function RequestedAmount({
             >
               <Checkbox checked={stable} onChange={onIsStableChange} />
               <span>Stable amount ({stableToken.symbol})</span>
+              {stable && (
```
Comment (0xGabi) [inline src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js:396]: I wonder if we should always show the Help tip.

File: src/components/Garden/ProposalDetail/ProposalDetail.js [hunk 2]
```diff
@@ -622,7 +622,16 @@ const Amount = ({
                 {requestToken.symbol}
               </span>
               <Help hint="">
-                Converted to {requestToken.symbol} at time of execution
+                Converted to {requestToken.symbol} at time of execution. For
```
Comment (0xGabi) [inline src/components/Garden/ProposalDetail/ProposalDetail.js:625]: Probably we should add the same message to the `ProposalInfo`. What do you think?

**Review by 0xGabi — CHANGES_REQUESTED** [review]

File: src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js (hunk 1 above)
Comment (fabriziovigevani) [reply src/components/Garden/ModalFlows/CreateProposalScreens/AddProposal.js:396]: Let´s do it

File: src/components/Garden/ProposalDetail/ProposalDetail.js (hunk 2 above)
Comment (fabriziovigevani) [reply src/components/Garden/ProposalDetail/ProposalDetail.js:625]: Ah yeas good catch

**Review by 0xGabi — APPROVED** [review]

**Pull request merged**